include/boost/histogram/accumulators/fraction.hpp

```diff
@@ -36,7 +37,8 @@ class fraction {
   using const_reference = const value_type&;
   using real_type = typename std::conditional<std::is_floating_point<value_type>::value,
                                               value_type, double>::type;
-  using interval_type = typename utility::wilson_interval<real_type>::interval_type;
+  using score_type = typename utility::wilson_interval<real_type>;
```

Comment:
score_type should not appear here, because it is not a type that needs to be publicly visible.

Reply:
This issue should be addressed now. Please take a look.